addon/model.js

```diff
@@ -54,7 +54,12 @@ function resolveValue(key, value, modelName, store, schema, model) {
     return resolvePlainArray(key, value, modelName, store, schema, model);
   }
 
-  let reference = schema.computeAttributeReference(key, value, modelName);
+  let reference = schema.computeAttributeReference(
```

Comment:
looks great; for consistency, can we pass `_data` to the other APIs as well?

Reply:
I added data hash as part other APIs `isAttributeArrayReference` and `computeNestedModel`